src/Meshes/instancedMesh.ts

```diff
@@ -70,7 +70,7 @@ export class InstancedMesh extends AbstractMesh {
 
         this.setPivotMatrix(source.getPivotMatrix());
 
-        this.refreshBoundingInfo();
+        this.refreshBoundingInfo(true, true);

```

Comment:
This one is a breaking change but I don't know if it should be noted in the what's new file...

Reply:
We considered it a bug fix :) The bb were wrong before

Reply:
I'll add it to what's new. I will also note that skeletons are now being set when cloned.